src/default-xrp-client.ts

```diff
@@ -1,8 +1,10 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
what is the restricted syntax?

Reply:
It was keeping me from using a for loop instead of array iteration via a map or something.  but I honestly vote for the for loop here.  Hans showed me how I could use flatmap but it was more confusing to look at and not any more performant

Reply:
(he suggested I just disable the linter for this case)

Reply:
Makes sense. Maps / flat maps are hard until you've looked at them long enough and I agree it's not exactly self explanatory so I'm happy to use more explicit syntax. 

BTW, I'd generally assume that nearly everything we do in this library is going to be performance bound by network latency so I wouldn't worry too much about performance of actual code unless you're doing something really egregious. Even then we  can always refactor later on.